### DISCUSSION TITLE
Using a wide gauge or a tall gauge stat card in an expander or a dialog

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1968] [Q&A] [answered]
Hi!So I am trying to implement a dialog or an expander, such that it displays statistics in the lines of

"30% of users who bought X also bought Y and Z"

Now I thought of implementing this using a gauge card along with ui.text() within a closable dialog or an expander. Is that possible? I got as far is creating the dialog box but I'm unable to add the stat_card within it. Is there a similar way to display a progress gauge or a percent gauge?
If not, any ideas on how I can implement this?

**Comment by mturoci** [comment, marked as answer]
Dialogs usually do not provide critical info and have a rather supportive function like confirmations, loading status etc. Wave doesn't allow displaying cards within dialogs.

If you are trying to display important information, a dedicated detail page might be a better option.

**Reply by n-srinidhi** [reply]
Got it! This makes sense. While I see its not possible to include `stat_card` inside a form, Is there a way to include a progress gauge inside a `ui.stats()` within a form_card?

**Reply by mturoci** [reply]
Gauge is not supported this time - https://wave.h2o.ai/docs/widgets/form/stats/.

**Reply by n-srinidhi** [reply]
Ok.Thank you!
